src/NHSD.BuyingCatalogue.Ordering.Api/Models/Summary/SectionStatusRequestModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models.Summary
+{
+    public class SectionStatusRequestModel
+    {
+        public string Status { get; set; }
```

Comment:
Also this class didnt need to be added as it was created in the canned data: 'UpdateOrderSectionModel"